### PR TITLE
Add structure reasons

### PR DESCRIPTION
**Pull request by anapaulavnoronha**

### CODE REVIEW

**Comment by anapaulavnoronha** [comment]
@caepenna @wkurosawa @carlosshaka

File: README.md [hunk 1]
```diff
@@ -1 +1,22 @@
...
+Beside that, our navigation structure is inside the header.
+Users are more used to find navigation link on the header, so having the navigation menu on header makes it more familiar to them.
+
+## CSS Structure
+
+The navigation has the most popular structure, is horizontal and across the top.
+
+## JS Structure
+
+To simplify, we tried to use less JS as possible in the structure.
```
Comment (caepenna) [inline README.md]: "as little js as possible"

File: README.md [hunk 2]
```diff
@@ -1 +1,22 @@
...
+We created a basic structure of responsive navigation to be reused and customized in different projects.
+
+## HTML Structure
+
+It contains the `nav` tag and you can add `a` tags inside it as you need.
+
+We decided to not use the list structure (`ul`>`li`>`a`) inside the nav tag.
+It makes the code cleaner, is semantic and reduce the amount of information for screen readers to read.
+
+Beside that, our navigation structure is inside the header.
```
Comment (caepenna) [inline README.md]: besides

File: README.md [hunk 3]
```diff
@@ -1 +1,22 @@
...
+Users are more used to find navigation link on the header, so having the navigation menu on header makes it more familiar to them.
```
Comment (caepenna) [inline README.md]: links

File: README.md [hunk 4]
```diff
@@ -1 +1,22 @@
...
+
+## CSS Structure
+
+The navigation has the most popular structure, is horizontal and across the top.
```
Comment (caepenna) [inline README.md]: Maybe add "But it can be customizes id necessary"?

**Pull request merged**